box_rules/box_item_shared_externally.yml

```diff
@@ -0,0 +1,52 @@
+AnalysisType: rule
+Filename: box_item_shared_externally.py
+RuleID: Box.Item.Shared.Externally
+DisplayName: Box item shared externally
+Enabled: false
+LogTypes:
+  - Box.Event
+Tags:
+  - Box
+Severity: Low
+Description: >
+  A user has shared an item and it is accessible to anyone with the share link (internal or external to the company). 
+Reference: https://developer.box.com/reference/resources/event/
+Runbook: >
+  Investigate whether this user's activity is expected. 
+SummaryAttributes:
+  - ip_address
```

Comment:
The `event['created_by']['name']`, `event['source']['item_name']` seem like useful attributes -- are we able to reference them here?

Reply:
Can we access nested fields now? I did have a hard time coming up with some summary attributes here that weren't nested

Reply:
I'm pretty  sure you can't reference nested fields sadly, something we're working on.